cli/src/commands/git.rs

```diff
@@ -220,7 +220,7 @@ pub struct GitPushArgs {
     /// correspond to missing local branches.
     #[arg(long)]
     deleted: bool,
-    /// Push branches that contain empty descriptions
+    /// Push branches with commits that contain empty descriptions
```

Comment:
nit: squash this commit into the previous one (see https://martinvonz.github.io/jj/v0.17.1/contributing/#code-reviews)

Reply:
Ah perfect. I prefer squashing and force pushing, but everyone is a bit different. I've squashed and pushed the changes! Let me know if you need anything else @martinvonz and I can take a look at it when I get the time.